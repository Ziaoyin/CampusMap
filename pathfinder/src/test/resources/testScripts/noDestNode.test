# tests that if given dest node doesn't exist, output is correct
CreateGraph graph
FindPath graph n1 n1

AddNode graph n1
FindPath graph n1 n2

AddNode graph n2
FindPath graph n2 n3