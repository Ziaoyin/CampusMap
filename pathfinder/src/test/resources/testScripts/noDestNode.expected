# tests that if given dest node doesn't exist, output is correct
created graph graph
unknown: n1

added node n1 to graph
unknown: n2

added node n2 to graph
unknown: n3